scripts/run_overnight_bench_smoke.py

```
@@ -0,0 +1,206 @@
+
+    meta = {
+        "started_at": _utc_now(),
+        "duration_hours": args.duration_hours,
+        "checkpoint_hours": args.checkpoint_hours,
+        "read_only": True,
+        "run_dir": str(run_dir),
+    }
+    (run_dir / "meta.json").write_text(json.dumps(meta, indent=2), encoding="utf-8")
+
+    if not args.skip_bootstrap:
+        subprocess.run([sys.executable, str(REPO / "scripts" / "bootstrap_bench_dual_source.py")], check=False)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)